jobs/e2e_node/perf-image-config.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 images:
   cos-stable1:
-    image: cos-81-12871-119-0 # docker 19.03.6, current latest LTS.
+    image_family: cos-85-lts # docker 19.03.9, current latest LTS.
```

Comment:
let's add deprecated after instead

Reply:
I am thinking to remove all the comments, WDYT?